src/main/java/uk/gov/pay/adminusers/resources/UserResource.java

```diff
@@ -79,25 +77,15 @@ public Response getUserByUsername(@QueryParam(USERNAME_FILTER_PARAMETER_KEY) Str
     @GET
     @Produces(APPLICATION_JSON)
     @Consumes(APPLICATION_JSON)
-    public Response getUser(@PathParam("externalId") String externalId, @QueryParam(IS_NEW_API_REQUEST_PARAMETER_KEY) String isNewApiRequest) {
+    public Response getUser(@PathParam("externalId") String externalId) {
         logger.info("User GET request - [ {} ]", externalId);
-        if (isNotBlank(isNewApiRequest)) {
-            if (isNotBlank(externalId) && (externalId.length() != USER_EXTERNAL_ID_LENGTH)) {
-                return Response.status(NOT_FOUND).build();
-            }
-
-            return userServices.findUserByExternalId(externalId)
-                    .map(user -> Response.status(OK).type(APPLICATION_JSON).entity(user).build())
-                    .orElseGet(() -> Response.status(NOT_FOUND).build());
-        } else {
-            if (isNotBlank(externalId) && (externalId.length() > USER_USERNAME_MAX_LENGTH)) {
-                return Response.status(NOT_FOUND).build();
-            }
-
-            return userServices.findUserByUsername(externalId)
-                    .map(user -> Response.status(OK).type(APPLICATION_JSON).entity(user).build())
-                    .orElseGet(() -> Response.status(NOT_FOUND).build());
+        if (isNotBlank(externalId) && (externalId.length() != USER_EXTERNAL_ID_LENGTH)) {
```

Comment:
As discussed I will remove these validations completely to be consistent with other endpoints like: `uk.gov.pay.adminusers.resources.UserResource#newSecondFactorPasscode`